packages/typespec-client-generator-core/src/index.ts

```diff
@@ -3,3 +3,4 @@ export * from "./interfaces.js";
 export * from "./lib.js";
 export * from "./public-utils.js";
 export * from "./types.js";
+export * from "./validate.js";
```

Comment:
```suggestion
export { $onValidate } from "./validate.js";
```

know you don't have precedent for that in this package but its good to try to only export what you need so you don't export somethihng meant private and breaking consumer by mistake